Benchmark/README.md

```diff
@@ -0,0 +1,33 @@
+## How to run the benchmarks
+
+The files in this folder define a benchmark suite with the tools provided by [PkgBenchmark](https://github.com/JuliaCI/PkgBenchmark.jl) and [BenchmarkTools](https://github.com/JuliaCI/BenchmarkTools.jl).
+
+To run the benchmarks, execute:
+
+```julia
+julia> using PkgBenchmark
+
+julia> results = benchmarkpkg("TaylorModels.jl")
```

Comment:
I think here the command should be `benchmarkpkg("TaylorModels")`; at least, with the extension "jl" as it appears here, Julia threw an error when I tested.